src/components/TodoUser/TodoUser.tsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect, useRef, useState } from 'react';
+import { Todo } from '../../types/Todo.js';
+import cn from 'classnames';
+
+type Props = {
+  todo: Todo;
+  onDelete: (id: number) => void;
+  processedId: number[];
+  handleCompletedStatus: (id: number) => void;
+};
+
+export const TodoUser: React.FC<Props> = ({
+  todo,
+  onDelete,
+  handleCompletedStatus,
+  processedId,
+}) => {
+  const [isEditingTodo, setIsEditingTodo] = useState<Todo | null>(null);
+
+  const { id, completed, title } = todo;
+
+  const processedTodos = processedId?.includes(id) || id === 0;
```

Comment:
```suggestion
  const isTodoProcessing = processedId?.includes(id) || id === 0;
```
It's a boolean value, not an array of todos
And processedId => processedIds